tests_integration/test_cp.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from unittest import mock
-from tests_integration.snowflake_connector import create_database, snowflake_session
+from tests_integration.snowflake_connector import database_name, snowflake_session
```

Comment:
IMO it may be confusing that you have to use "snowflake_session" fixture in your test case even if you don't want to execute SQLs in your test (you want to only use SnowCLI command requiring some database).